views/templates/panes/compiler.pug

```diff
@@ -22,6 +22,21 @@ mixin newPaneButton(classId, text, title, icon)
               .argmenuitem
                 span.argtitle Detailed Compiler Flags
                 span.argdescription Open a new window to edit verbose compiler flags
+    .btn-group.btn-group-sm.w-100(role="group" aria-label="Compiler revision picker")
+      noscript This feature requires JavaScript to be enabled.
+      .input-group.compiler-revision-picker
+        <!-- Selector to be wrapped in TomSelect -->
```

Comment:
These comments aren't really helping I think - I'm not against them but no other controls have this (it's kinda accepted that a `select` widget will be for example skinned with `TomSelect`, or whatever system we choose). These have a tendency to rot, as they're a long way from the code that actually controls them.

Reply:
Done